src/services/userServices.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+let users = [];
+
+const getAllUsers = () => users;
+
+const getUserById = (id) => {
+  return users.find(user => user.id === +id) || null;
```

Comment:
<img width="781" alt="image" src="https://github.com/mate-academy/node_accounting-app-v2/assets/54860641/5573b8a8-06ca-487f-bb41-b1d60c2e9e99">

```suggestion
  return users.find(user => user.id === +id);
```